ios/Peepz/Peepz/PeepzApp.swift

```diff
@@ -18,7 +18,7 @@ class AppDelegate: NSObject, UIApplicationDelegate {
 @main
 struct PeepzApp: App {
     @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
-    @StateObject private var model = PeepzModel()
+    @StateObject private var model: PeepzModel = .live
```

Comment:
Here we can simply change to `.mock` to use static data instead and skip the Google Login

Reply:
could we trigger this from app settings when in dev? that would be a cool addition

Reply:
yes, I think it's definitely something that would be worth to investigate 👍